src/platform/plugins/shared/chart_expressions/expression_heatmap/public/expression_renderers/heatmap_renderer.tsx

```diff
@@ -62,6 +62,13 @@ export const heatmapRenderer: (
   }),
   reuseDomNode: true,
   render: async (domNode, config, handlers) => {
+    const performanceTracker = createPerformanceTracker({
+      type: 'Lens',
```

Comment:
Could be nice to make `Lens` a shared const so we don't mix up upper and lower cases